app/views/subscription/confirm.html.erb

```diff
@@ -1,12 +1,18 @@
 <%= form_tag url: confirm_url do %>
 
-  <div class='form-group'>
-    <%= label_tag :confirmation_code  %>
-    <%= text_field_tag :confirmation_code, '', class: 'form-control' %>
-  </div>
+<div class="container">
+	<div class="show-single-result">
```

Comment:
@Tallys It seems like this class is no longer semantic - this should be something generalized to work for both notions.